src/libos/src/fs/fs_ops/mount.rs

```diff
@@ -27,5 +28,13 @@ pub fn do_mount_rootfs(
         *root_inode = new_root_inode;
         *ENTRY_POINTS.write().unwrap() = user_config.entry_points.to_owned();
     });
+    let fs_view = FsView::new();
+    let resolv_conf_file = match fs_view.open_file("/etc/resolv.conf", AccessMode::O_RDWR as u32 | CreationFlags::O_CREAT.bits(), 0o666){
```

Comment:
Should check whether the file is already there? If the user copied the file, then we should do nothing.

Reply:
Ok， I will consider this situation.

Reply:
Done